### DISCUSSION TITLE
auto prettify the flow blocks

### DISCUSSION THREAD

**Discussion by ironmanoyo** [postmanlabs/postman-flows#84] [Ideas]
It would be great if there's was a way to auto prettify all blocks or selected blocks using a keyboard shortcut.
In future, it can be evolved to **Auto-Simplify**

**Comment by ironmanoyo** [comment]
@samay-v 

I was thinking of an auto pretty in the below manner. (Clumsy flow/flowchart to readable flow/flowchart)
The below video was taken from the lucidchart

https://user-images.githubusercontent.com/45940611/139419249-7faa5bd7-4465-4bba-85d2-b329ad994372.mov

**Reply by saswatds** [reply]
Hey @ironmanoyo, thanks for the feedback. I have taken note of this. 
We had tried to implement auto format with `dagre` before, but it did not turn out to be that good. If you come across any other algorithm do let us know, meanwhile we will also keep looking or else we will implement our own.
